programs/bpf-loader-tests/tests/common.rs

```diff
@@ -14,7 +15,9 @@ use {
 };
 
 pub async fn setup_test_context() -> ProgramTestContext {
-    let program_test = ProgramTest::new("", id(), Some(solana_bpf_loader_program::Entrypoint::vm));
+    let mut program_test =
+        ProgramTest::new("", id(), Some(solana_bpf_loader_program::Entrypoint::vm));
+    program_test.deactivate_feature(enable_extend_program_checked::id());
```

Comment:
Could you fixup these tests to use the new instruction always? It shouldn't be too much extra work, or I can help out on that if you like